src/Footer.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const props = [
+    "123 Fake Street, London, E1 4UD", 
+    "hello@fakehotel.com", 
+    "0123 456789"
+];
+
+
+const Footer = () => {
+  return (
+    <footer>
+      <ul className="footer-contact">
+        {props.map((props) => (
```

Comment:
The variable name `props` is reserved for the argument passed to all React components (e.g. `const Footer = (props) => {...}`. Try not to call other variables `props` except for that one specific case, as it can be confusing. In this case, if you want to have default arguments for your props, you could use this syntax:
```
const Footer = (props = {address: "123 Fake Street", email: "hello@fakehotel.com", phoneNumber: "0123 456789"}) => {...}
```

or

```
const defaultProps =  {address: "123 Fake Street", email: "hello@fakehotel.com", phoneNumber: "0123 456789"};

const Footer = (props = defaultProps) => {...}
```